app/routes.py

```diff
@@ -0,0 +1,357 @@
+from app import db
+from flask import Blueprint, jsonify, request
+from datetime import datetime, timedelta
+from app.models.video import Video
+from app.models.customer import Customer
+from app.models.rental import Rental
+
+
+videos_bp = Blueprint("videos", __name__, url_prefix=("/videos"))
+
+#POST/CREATE 
+@videos_bp.route("",methods=["POST"])
+def post_videos():
+    request_body = request.get_json()
+    if "title" not in request_body:
+        return{
+            "details": "Request body must include title."
+        }, 400
+    
+    elif "release_date" not in request_body:
+        return{
+            "details": "Request body must include release_date."
+        }, 400
+    
+    elif "total_inventory" not in request_body:
+        return{
+            "details": "Request body must include total_inventory."
+        }, 400
```

Comment:
Since the outcome of any of these conditions being true is for the function to exit, these could all be `if` rather than `elif`. More interestingly, notice how each of these conditions is more or less the same. They differ only in the key being checked for in the request, and the string being used in the error message. If we put the keys into a list and iterated over it, how could we rewrite this to avoid duplicating the logic three times (or potentially more if there were other keys we needed to check!)?